mitosheet/mitosheet/parser.py

```diff
@@ -656,48 +717,47 @@ def get_header_index_header_index_match(formula: str, raw_parser_matches: List[R
         formula,
         formula_label,
         string_matches,
-        df
+        dfs,
+        df_names,
+        sheet_index
     )
 
     match_index = 0
     while match_index < len(raw_parser_matches):
-
-        header_index_header_index_match = get_header_index_header_index_match(formula, raw_parser_matches, match_index)
-        if header_index_header_index_match is not None:
+        if (header_index_header_index_match := get_header_index_header_index_match(formula, raw_parser_matches, match_index)) is not None:
```

Comment:
The := syntax doesn't work on Python 3.6 -- you'll sadly have to define these on a separate line </3. This might be why CI is failing.